### PR TITLE
Ft search

### PR DESCRIPTION
**Pull request by cangfengzhs**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by critical27 — APPROVED** [review]
It seems some tck using fulltext index failed, test_fulltext_ddl.

File: src/common/plugin/fulltext/elasticsearch/ESAdapter.cpp [hunk 1]
```diff
@@ -68,8 +65,10 @@ void ESAdapter::setClients(std::vector<ESClient>&& clients) {
   clients_ = std::move(clients);
 }
 
-Status ESAdapter::createIndex(const std::string& name) {
-  folly::dynamic mappings = folly::parseJson(R"(
+Status ESAdapter::createIndex(const std::string& name,
+                              const std::vector<std::string>& fields,
+                              const std::string& analyzer) {
```
Comment (critical27) [inline src/common/plugin/fulltext/elasticsearch/ESAdapter.cpp:70]: What is analyzer here? 分词器？

Comment (cangfengzhs) [reply src/common/plugin/fulltext/elasticsearch/ESAdapter.cpp:70]: Yes

File: tests/tck/features/fulltext_index/FulltextIndexQuery1.feature [hunk 2]
```diff
@@ -1,107 +0,0 @@
-# Copyright (c) 2022 vesoft inc. All rights reserved.
-#
```
Comment (yixinglu) [inline tests/tck/features/fulltext_index/FulltextIndexQuery1.feature:2]: Why delete these tests? i add more cases in #5575 in these features.

Comment (cangfengzhs) [reply tests/tck/features/fulltext_index/FulltextIndexQuery1.feature:2]: Because PREFIX,FUZZY,WILDCARD,REGEXP are all abandoned

**Review by yixinglu — APPROVED** [review]
LGTM

**Review by critical27 — APPROVED** [review]

**Pull request merged**